frontend/src/api-service/vegetationCodeAPI.ts

```diff
@@ -0,0 +1,22 @@
+import ApiConfig from './ApiConfig';
+import api from './api';
+
+const getVegCodes = () => {
+  const url = ApiConfig.vegetationCode;
+  return api.get(url).then((res) => {
+    const vegCodeOptions: Array<string> = [];
+    if (res.data) {
+      res.data.forEach((vegCode: any) => {
```

Comment:
Since we know the data and props from this endpoint, could we create an interface and avoid using `any`?

Reply:
[added](https://github.com/bcgov/nr-spar-webapp/pull/85/commits/ad6bc57d4c6c68fddb11fedfb873e0b752a361da)